src/vignette/util/consul.clj

```diff
@@ -0,0 +1,49 @@
+(set! *warn-on-reflection* true)
```

Comment:
why is this needed? does it change it for the running service as a whole, or just for this namespace?